nixos/modules/programs/sway.nix

```diff
@@ -1,41 +1,36 @@
-{ config
-, pkgs
-, lib
-, ... }:
+{ config, pkgs, lib, ... }:
 
 with lib;
 
 let
   cfg = config.programs.sway;
 
   wrapperOptions = types.submodule {
-    options = {
-      base = mkOption {
-        type = types.bool;
-        default = true;
-        example = "false";
-        description = ''
-          Make use of the base wrapper to execute extra session commands
-          and prepend a dbus-run-session to the sway command.
+    options =
+      let
+        mkWrapperFeature  = default: description: mkOption {
```

Comment:
Thought this could be useful to avoid mistakes and redundancy. Also e.g. `example = "false";` didn't need any quoting.

Reply:
This is really clean, I didn't know it could be written this way, thanks for showing it :)